apps/proxy/lib/proxy/routes/api.ex

```diff
@@ -32,7 +32,17 @@ defmodule Proxy.Routes.API do
   post "/system/:name/actors/:actor_name/invoke" do
     with %InvocationRequest{system: sytem, actor: actor} = request <-
            get_body(conn.body_params, InvocationRequest),
-         {:ok, %ActorInvocationResponse{value: value} = _response} <- Actors.invoke(request) do
+         response <- Actors.invoke(request) do
```

Comment:
Was erroring due to always expecting {:ok, term} tuple, when it was async, due to being a GenServer.cast, it only returned :ok